index.html

```diff
@@ -0,0 +1,147 @@
+      <h3>And I'm <span>Junior Web Developer</span></h3>
+      <p> I'm Mehria Saqibi, a passionate junior web developer with a Computer Science degree. I thrive on turning complex problems into elegant web solutions.
+      </p>
+      <div class="social-media">
+        <a href="#"><i class='bx bxl-facebook'></i></a>
+        <a href="#"><i class='bx bxl-twitter'></i></a>
+        <a href="#"><i class='bx bxl-instagram'></i></a>
+        <a href="#"><i class='bx bxl-linkedin-square'></i></a>
+      </div>
+      <a href="#" class="btn">Download CV</a>
+    </div>
+    <div class="home-img">
+      <img src="img/girl-programmer.png" alt="">
+    </div>
+  </section>
+
+  <!-- about section design -->
+  <section class="about" id="about">
+    <div class="about-img">
+      <img src="img/girl-programmer.png" alt="">
+    </div>
+
+    <div class="about-content">
+      <h2 class="heading">About <span>Me</span></h2>
+      <h3>Junior Developer!</h3>
+      <p>I'm Mehria Saqibi, a junior web developer with a Computer Science degree. With two years of teaching experience, one year as a consultant in an NGO, and a year as a translator for RUMIE BUILD VOLUNTEER, I bring a diverse skill set including web development, problem-solving, and communication. I thrive on creating elegant web solutions and contributing to meaningful projects."</p>
+      <a href="#" class="btn">Read More</a>
+    </div>
+  </section>
+  <!-- services section design -->
+  <section class="services" id="services">
+    <h2 class="heading">Our <span>Services</span></h2>
+    <div class="services-container">
+      <div class="services-box">
+        <i class='bx bx-paint'></i>
+        <h3>Gaphic Design</h3>
+        <p>We offer comprehensive services in web development, graphic design, and digital marketing. As junior web developers, we create dynamic, user-friendly websites, design captivating graphics, and implement effective digital marketing strategies to help your business thrive online.</p>
+        <a href="#" class="btn">Read More</a>
+      </div>
+      <div class="services-box">
+        <i class='bx bx-code-alt'></i>
+        <h3>Web Development</h3>
+        <p>We offer comprehensive services in web development, graphic design, and digital marketing. As junior web developers, we create dynamic, user-friendly websites, design captivating graphics, and implement effective digital marketing strategies to help your business thrive online."</p>
+        <a href="#" class="btn">Read More</a>
+      </div>
+      <div class="services-box">
+        <i class='bx bx-bar-chart-alt-2'></i>
+        <h3>Digital Marketing</h3>
+        <p>We offer comprehensive services in web development, graphic design, and digital marketing. As junior web developers, we create dynamic, user-friendly websites, design captivating graphics, and implement effective digital marketing strategies to help your business thrive online."</p>
+        <a href="#" class="btn">Read More</a>
+      </div>
```

Comment:
![Screenshot 2024-06-06 124419](https://github.com/Saqibi4213/my-portfolio/assets/96838030/b3e2032b-21e7-4461-8b0b-3b1bfbf4cb87)

- Try to add the actual and real data about your services here and also if there is any link for your services add the link (a) tag other than that you can remove them.